src/components/Header/Header.js

```diff
@@ -19,7 +19,13 @@ function Header() {
   const network = getNetwork()
   const { connectedGarden } = useGardens()
 
-  const BeeIcon = <img src={beeSvg} height={layoutSmall ? 40 : 60} alt="" />
+  const logo =
+    connectedGarden && connectedGarden.logo ? connectedGarden.logo : beeSvg
+  const logoType =
+    connectedGarden && connectedGarden.logo_type
+      ? connectedGarden.logo_type
+      : logoSvg
+  const BeeIcon = <img src={logo} height={layoutSmall ? 40 : 60} alt="" />
```

Comment:
Let's perhaps rename to something more general now
```suggestion
  const Logo = <img src={logo} height={layoutSmall ? 40 : 60} alt="" />
```